### ISSUE TITLE
Deprecation message

### ISSUE THREAD

**Issue by haivala** [Arkounay/ux-collection#7]
```
deprecation.INFO: User Deprecated: Method "Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface::prepend()" might add "void" as a native return type declaration in the future. Do the same in implementation "Arkounay\Bundle\UxCollectionBundle\DependencyInjection\ArkounayUxCollectionExtension" now to avoid errors or add an explicit @return annotation to suppress this message.
```

**Comment by haivala** [comment]
another one
`Method "Symfony\Component\DependencyInjection\Extension\ExtensionInterface::load()" might add "void" as a native return type declaration in the future. Do the same in implementation "Arkounay\Bundle\UxCollectionBundle\DependencyInjection\ArkounayUxCollectionExtension" now to avoid errors or add an explicit @return annotation to suppress this message.`

**Comment by Arkounay** [comment]
Thanks, this should be fixed in [3.3.3](https://github.com/Arkounay/ux-collection/releases/tag/3.3.3)
